pkg/daemon/update.go

```diff
@@ -1937,7 +1937,7 @@ func (dn *Daemon) InplaceUpdateViaNewContainer(target string) error {
 		glog.Info("SELinux is not enforcing")
 	}
 
-	err = runCmdSync("systemd-run", "--unit", "machine-config-daemon-update-rpmostree-via-container", "--collect", "--wait", "--", "podman", "run", "--env-file", "/etc/mco/proxy.env", "--authfile", "/var/lib/kubelet/config.json", "--privileged", "--pid=host", "--net=host", "--rm", "-v", "/:/run/host", target, "rpm-ostree", "ex", "deploy-from-self", "/run/host")
+	err = runCmdSync("systemd-run", "--unit", "machine-config-daemon-update-rpmostree-via-container", "-p", "EnvironmentFile=-/etc/mco/proxy.env", "--collect", "--wait", "--", "podman", "run", "--env-file", "/etc/mco/proxy.env", "--authfile", "/var/lib/kubelet/config.json", "--privileged", "--pid=host", "--net=host", "--rm", "-v", "/:/run/host", target, "rpm-ostree", "ex", "deploy-from-self", "/run/host")
```

Comment:
Looks like there is an extra "-" before /etc/mco/proxy.env

Reply:
Yeah, that's intentional because that's the systemd way to say "it's OK if this file doesn't exist".  Today we write an *empty* `/etc/mco/proxy.env` if there's no proxy; I think it'd be cleaner sometime in the future to stop doing that.  See https://www.freedesktop.org/software/systemd/man/systemd.exec.html#EnvironmentFile=